react/react1/week2/todo-app/src/components/TodoList.js

```diff
@@ -0,0 +1,41 @@
+import { useState } from 'react';
+import { todos } from '../data';
+import Button from './Button';
+import TodoRow from './TodoRow';
+
+const TodoList = () => {
+  const [todosList, setTodosList] = useState(todos);
+
+  const newTodo = {
+    id: todosList.length + 1,
```

Comment:
I'm not sure if you meant to have an unique id, which I guess you should. This way will give you repetitive Ids, by just adding and removing and adding again. a better solution to generate a random unique ID can be this : `Math.random().toString(16).slice(2)`